cardano-node/src/Cardano/Node/Tracing/Era/Shelley.hs

```diff
@@ -306,22 +309,22 @@ renderScriptIntegrityHash Nothing = Aeson.Null
 renderScriptHash :: ScriptHash StandardCrypto -> Text
 renderScriptHash = Api.serialiseToRawBytesHexText . Api.fromShelleyScriptHash
 
-renderMissingRedeemers :: [(Alonzo.ScriptPurpose StandardCrypto, ScriptHash StandardCrypto)] -> Aeson.Value
-renderMissingRedeemers scripts = Aeson.object $ map renderTuple  scripts
+renderMissingRedeemers :: Typeable era => Proxy era -> [(Alonzo.ScriptPurpose StandardCrypto, ScriptHash StandardCrypto)] -> Aeson.Value
```

Comment:
This can also work without proxy and with `AllowAmbiguousTypes`:
```haskell
renderMissingRedeemers :: Typeable era => [(Alonzo.ScriptPurpose StandardCrypto, ScriptHash StandardCrypto)] -> Aeson.Value
```
each call site would require type annotation though:
```haskell
renderMissingRedeemers @ShelleyEra []
```